.github/workflows/ci.yml

```diff
@@ -151,39 +144,13 @@ jobs:
           
           $connectionString = "Data Source=localhost;Initial Catalog=nservicebus;User ID=particular;Password=Welcome1;AllowUserVariables=True;AutoEnlist=false;Connect Timeout=60"
           echo "MySQLConnectionString=$connectionString" | Out-File -FilePath $Env:GITHUB_ENV -Encoding utf-8 -Append
-
-      - name: Setup Oracle
+      - name: Install Oracle
         id: setup-oracle
         if: matrix.engine == 'Oracle'
-        shell: pwsh
-        run: |
-          if ( $Env:RUNNER_OS -eq 'Windows') {
-            echo "Adding environment variables for Oracle connection strings"
-            echo "OracleConnectionString=User Id=particular;Password=Welcome1;Data Source=127.0.0.1:1521/nservicebus;Enlist=dynamic" | Out-File -FilePath $Env:GITHUB_ENV -Encoding utf-8 -Append
-            echo "OracleConnectionString_Particular2=User Id=particular2;Password=Welcome1;Data Source=127.0.0.1:1521/nservicebus;Enlist=dynamic" | Out-File -FilePath $Env:GITHUB_ENV -Encoding utf-8 -Append
-          }
-
-          if ( $Env:RUNNER_OS -eq 'Linux' ) {
-            echo "Running Oracle using Docker"
-            docker run --name oracle -d -p 1521:1521 -e ORACLE_PASSWORD=Welcome1 gvenzl/oracle-xe:21
-
-            for ($i = 0; $i -lt 24; $i++) { ## 2 minute timeout
-              echo "Checking for Oracle connectivity $($i+1)/24..."
-              docker exec oracle ./healthcheck.sh
-              if ($?) {
-                echo "Connection successful"
-                break;
-              }
-              sleep 5
-            }
-
-            cat ./.github/workflows/scripts/oracle-setup-linux.sql | docker exec -i oracle sqlplus / as sysdba
```

Comment:
And also make sure to execute this

Reply:
We have to add this ability to the setup action, but it should be fairly straightforward since we can just pipe through the scripts into docker or ACI. I can add that tomorrow.